.github/workflows/chromatic.yml

```diff
@@ -109,5 +109,8 @@ jobs:
 
             ### [Endringer til review](${{steps.chromatic_tests.outputs.url}}): ${{steps.chromatic_tests.outputs.changeCount}}
 
+            ${{ steps.chromatic_tests.outputs.errorCount > 0 && format('### ❌ Stories med feil: {0}', steps.chromatic_tests.outputs.errorCount) || '' }}
+            ${{ steps.chromatic_tests.outputs.interactionTestFailuresCount > 0 && format('### ❌ Feilede interaksjonstester: {0}', steps.chromatic_tests.outputs.interactionTestFailuresCount) || '' }}
```

Comment:
Haven't been able to figure out what will cause `interactionTestFailuresCount` to be > 0 🤔

Reply:
Will it not be be these tests that causes `interactionTestFailuresCount` to work?

<img width="1206" alt="Screenshot 2024-10-31 at 12 11 15" src="https://github.com/user-attachments/assets/05628a64-fb4f-41a5-944f-644b8542f359">

Reply:
You would think that, but everything I tried only affected `errorCount`. Look at the commits to see what I tried.